src/literals.py

```diff
@@ -287,11 +288,17 @@ class Status(Enum):
     )
 
 
+class KRaftUnitStatus(Enum):
```

Comment:
**question** Not sure what is this used for? It seems to me to be used only in tests. If so, we could maybe move it to tests instead of in the core codebase

Reply:
You're right it's only used in tests, moved it to `tests/integration/helpers.py`